app/views/api/v1/organizations/_organization.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.organization organization, :id, :name, :default_currency
```

Comment:
@yunixon it's better not to include `organization: ...` in json in this partial. In this case you can't use this parial to render collection. If you want then put `organization: ...` in upper level view.